src/arch/Sound/DSoundHelpers.h

```diff
@@ -31,6 +31,8 @@ class DSound
 class DSoundBuf
 {
 public:
+	static constexpr int KB_512_AS_BYTES = 524288; /* 512k for setting the buffer */
```

Comment:
I prefer the 512 * 1024 method. Let's also to 512_KB instead of KB_512

Reply:
No worries, I'll put it back like that. (However the name of a constant can't start with a number hence KB_512)